cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -109,12 +109,14 @@ func (command *Evaluate) Execute(args []string) (err error) {
 
 	// Check that models and languages can be evaluated by executing the "plain" repositories.
 	log.Printf("Checking that models and languages can be used for evaluation")
-	assessmentsPerModel := map[string]metrics.Assessments{}
+	assessmentsPerModelPerRepository := map[string]map[string]metrics.Assessments{}
```

Comment:
Should we introduce identifier types for models, languages, ... ? Would make them easier to read and allows to not mix? Kind ... of ... type ... safety?

Reply:
Could also use the models and languages directly instead of IDs. That way its type safe already.

Reply:
i mean more in the sense of something like this

someMap[modelID]...

would then fail if the key of the map is of type repositoryID

Reply:
Could also work. But why introduce extra types if we could just do:

```go
// AssessmentPerModelPerLanguagePerRepository holds a collection of assessments per model per language and per repository.
type AssessmentPerModelPerLanguagePerRepository map[model.Model]map[language.Language]map[string]Assessments
```

Reply:
Sure!